QuizApp/QuizApp/UseCase/UseCaseModels.swift

```diff
@@ -0,0 +1,77 @@
+struct LoginResponseModel {
+
+    let accessToken: String
+
+}
+
+extension LoginResponseModel {
+
+    init(_ loginResponse: LoginResponseRepoModel) {
+        accessToken = loginResponse.accessToken
+    }
+
+}
+
+struct UserInfoModel {
+
+    let username: String
+    let name: String
+
+}
+
+extension UserInfoModel {
+
+    init(_ userInfo: UserInfo) {
```

Comment:
Is UserInfo part of UI layer? If so, then this `init` should not be in the UC layer.